examples2/src/counter_pack.rs

```diff
@@ -5,6 +5,7 @@ use odra::Mapping;
 use odra::Module;
 use odra::ModuleWrapper;
 
+#[odra_macros::module]
```

Comment:
Why not `odra::module`?

Reply:
Will be, must be reexported in `odra` first.

Reply:
You should do it in this PR.

Reply:
I could but it does not matter, the feature still is not completed, and there will be a few more prs related to macros, and the release branch is under development.